### PR TITLE
Fixed issue with accented character name

### PR DESCRIPTION
**Pull request by telunc**
When searching with an accented character name, API will return no result. This is because `request-promise` does not accept accented letters in uri. I fixed it by encoding accented components.

Example of the bug:
`GET /api/character/Moká` will return `{"error":"Could not find character"}`

### CODE REVIEW

File: src/models/character.js [hunk 1]
```diff
@@ -5,14 +5,16 @@ import cacheManager from 'cache-manager'
 import redisStore from 'cache-manager-redis'
 import { ENV, PORT, DATADOG_API_KEY, DATADOG_APP_KEY, REDIS_HOST, REDIS_PORT } from '../environment'
 import FileSystem from 'fs'
+var Entities = require('html-entities').XmlEntities;
```
Comment (Inumedia) [inline src/models/character.js]: Why is this required? Doesn't seem like it's used below.

Comment (telunc) [inline src/models/character.js]: The library is used for these two lines. 
`var entities = new Entities()` 
`entities.decode(character.name)`

The html-entities library is used to decode html ASCII. When request-promise returning a character name with accent it will usually be in this form: `Mok&#225;`

Comment (Inumedia) [inline src/models/character.js]: Ah, I see it now.

**Comment by Inumedia** [comment]
This'll get deployed in the next few days.
